ocm_python_wrapper/cluster.py

```diff
@@ -249,33 +250,69 @@ def addon_info(self):
             self.addon_name
         ).to_dict()
 
-    def validate_addon_parameters(self, parameters):
+    def drop_unsupported_parameters(self, parameters=[]):
```

Comment:
do not use mutables as defaults (https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/)

Reply:
done